modular_skyrat/modules/opposing_force/code/opposing_force_equipment.dm

```diff
@@ -0,0 +1,572 @@
+	description = "An eight-round pump-action shotgun found in an old station. Comes loaded with beanbag shells but can take any 12 gauge load."
+
+/datum/opposing_force_equipment/gun/as2
+	item_type = /obj/item/gun/ballistic/shotgun/automatic/as2
+	description = "A compact version of the combat shotgun. Comes with a four-round tube and can be silenced as well as holstered to your belt or vest. \
+		Comes loaded with slugs but can take any 12 gauge load."
+
+/datum/opposing_force_equipment/gun/sas14
+	item_type = /obj/item/gun/ballistic/shotgun/sas14
+	description = "A semi-automatic, magazine-fed shotgun chambered in 14 gauge. Standard mags can take five rounds. \
+		Despite the lower damage of 14 gauge, it can load the highly praised taser shots."
+
+//here comes syndiecake
+
+/datum/opposing_force_equipment/gun/g357
+	item_type = /obj/item/gun/ballistic/revolver
+	description = "A .357 magnum revolver. Seven shots, more than enough to kill anything that moves."
+
+/datum/opposing_force_equipment/gun/mateba
+	item_type = /obj/item/gun/ballistic/revolver/mateba
+
+/datum/opposing_force_equipment/gun/nagant
+	item_type = /obj/item/gun/ballistic/revolver/nagant
+//lasguns
+
+//CFA Family
+
+/datum/opposing_force_equipment/gun/snub
+	item_type = /obj/item/gun/ballistic/automatic/pistol/cfa_snub
+
+/datum/opposing_force_equipment/gun/ruby
+	item_type = /obj/item/gun/ballistic/automatic/pistol/cfa_ruby
+
+/datum/opposing_force_equipment/gun/wildcat
+	item_type = /obj/item/gun/ballistic/automatic/cfa_wildcat
+
+//oddities
+/datum/opposing_force_equipment/gun/ninjastar
+	item_type = /obj/item/throwing_star
+	description = "Be the maintenance ninja you always wanted to be. Does not come with multi-throwing cybernetics"
+
+
+/datum/opposing_force_equipment/gun/throwing_weapons
+	item_type = /obj/item/storage/box/syndie_kit/throwing_weapons
+	description = "A box of shurikens and reinforced bolas from ancient Earth martial arts. They are highly effective \
+			throwing weapons. The bolas can knock a target down and the shurikens will embed into limbs."
+	name = "Box of Throwing Weapons"
+
+/datum/opposing_force_equipment/gun/origami
+	item_type = /obj/item/storage/box/syndie_kit/origami_bundle
+	description = "A box containing a guide on how to craft masterful works of origami, allowing you to transform normal pieces of paper into \
+			perfectly aerodynamic (and potentially lethal) paper airplanes."
+
+
+
+/**
+ * AMMO
+ */
+/datum/opposing_force_equipment/ammo/
+	category = OPFOR_EQUIPMENT_CATEGORY_AMMUNITION
+
+/datum/opposing_force_equipment/ammo/m45
+	item_type = /obj/item/ammo_box/magazine/m45
+	description = "An eight-round magazine for the M1911 pistol. Uses .45 caliber ammunition."
+
+/datum/opposing_force_equipment/ammo/m16
+	item_type = /obj/item/ammo_box/magazine/m16
+	description = "A twenty-round magazine for the M16 assault rifle. Uses 5.56x45mm ammunition."
+
+/datum/opposing_force_equipment/ammo/akm
```

Comment:
I don't think making the AKM available to the OPFOR people is a good idea, it's a ridiculously unbalanced gun.